advanced/PnID parsing using SDK.ipynb

```diff
@@ -141,12 +297,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Start PnID parsing job\n",
-    "job = client.pnid_parsing.parse(file_id=f.id, entities=entities)"
+    "job = client.pnid_parsing.detect(file_id=f.id, entities=entities)"
```

Comment:
As above, maybe add that file_external_id can be used instead of file_id?

Reply:
yes